recipe/meta.yaml

```
@@ -41,6 +41,8 @@ requirements:
     - {{ compiler('cuda') }}  # [linux64 or ppc64le or win]
     - {{ cdt('mesa-libgl-devel') }}  # [linux]
     - sysroot_linux-64 2.17  # [linux64]
+    - libcufft-dev  # [(linux64 or ppc64le or win) and cuda_compiler_version == "12.0"]
+    - cuda-profiler-api  # [(linux64 or ppc64le or win) and cuda_compiler_version == "12.0"]
```

This might be a more future proof strategy

```suggestion
    - libcufft-dev  # [(linux64 or ppc64le or win) and cuda_compiler_version.startswith("12")]
    - cuda-profiler-api  # [(linux64 or ppc64le or win) and cuda_compiler_version.startswith("12")]
```

---

Yesss, I was just thinking about how this will become a pain to fix later, thanks!